src/utils.js

```diff
@@ -0,0 +1,28 @@
+import { cloneDeep } from 'lodash';
+import { transformFromAstSync } from '@babel/core';
+import { ensureFileSync, writeFileSync } from 'fs-extra';
+
+function createSingleTestFiles (ast, describeIndex, itIndexes, spec) {
```

Comment:
instead of having one export at the end of the file

```suggestion
export function createSingleTestFiles (ast, describeIndex, itIndexes, spec) {
```

Reply:
👍